### PR TITLE
DONT MERGE YET: Fabric component: have text color theme

### PR DESCRIPTION
**Pull request by phkuo**
#### Pull request checklist

- [ ] Addresses an existing issue: Fixes #0000
- [x] Include a change request file using `$ npm run change`

#### Description of changes

Fabric component: have text color theme
Right now, the root Fabric component has both the .ms-Fabric class and the autogenerated class on it. Both define the color property. However, the .ms-Fabric class is not themable, but its specificity is equal to the autogenerated class. If it comes later in the page, its non-theming color will overwrite the themable color of the autogenerated class. The fix is to specifically override .ms-Fabric's color with something with higher specificity for the Fabric component.

#### Focus areas to test

(optional)

### CODE REVIEW

**Comment by phkuo** [comment]
Please don't merge this yet!

File: common/changes/office-ui-fabric-react/phkuo-msFabricColor_2017-08-15-18-14.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "packageName": "office-ui-fabric-react",
+      "comment": "Fabric component: have the text color theme",
```
Comment (dzearing) [inline common/changes/office-ui-fabric-react/phkuo-msFabricColor_2017-08-15-18-14.json]: This shows up in the public release notes...

Can you please be a lot more specific on what's changing?

File: packages/office-ui-fabric-react/src/components/Fabric/Fabric.styles.ts [hunk 2]
```diff
@@ -21,7 +21,8 @@ export const getStyles = memoizeFunction((
     root: mergeStyles([
       theme.fonts.medium,
       {
-        color: theme.palette.neutralPrimary,
+        color: theme.semanticColors.bodyText,
+        '&.ms-Fabric': { color: theme.semanticColors.bodyText }, // overwrite default ms-Fabric's color with one that themes
```
Comment (dzearing) [inline packages/office-ui-fabric-react/src/components/Fabric/Fabric.styles.ts]: this change doesn't make sense to me.

color is already assigned to this class. In addition, you're creating a selector (which is 2 levels of specificity deep) setting the color again.

What is the purpose of this?

**Comment by micahgodbolt** [comment]
@phkuo still hoping to push this through? If you aren't ready yet, close this PR up and we can come back to it when you're ready

**Pull request closed without merging**